Formula/fstrm.rb

```diff
@@ -33,7 +32,8 @@ def install
   end
 
   test do
-    system "sh", "-c", "#{bin}/fstrm_capture -t protobuf:dnstap.Dnstap -u dnstap.sock -w capture.fstrm -dddd & sleep 1; kill $!"
+    system "sh", "-c", "#{bin}/fstrm_capture -t protobuf:dnstap.Dnstap -u \
```

Comment:
The answer seemed to be no, calling `sh` explicitly is not a required step. I've made it more Ruby-ish.